pytodo/argparser.py

```diff
@@ -18,4 +18,5 @@ def create_parser():
     parser.add_argument("--edit", action="append", help="edit an existing todo")
     parser.add_argument("--clear", action="append", help="delete all todos")
     parser.add_argument("--expire", action="append", help="delete all overdue todos")
+    parser.add_argument("--disable_quotes", action="append", help="Disable/Enable quotes from displaying")
```

Comment:
It would be better to create a --quotes option, with 1 enabling tags and 0 disabling tags. --disable-quotes is a bit unintuitive.